### PR TITLE
♻️ Update Help.ts to improve maintainability

### PR DESCRIPTION
**Pull request by EdvinAldi**

### CODE REVIEW

File: src/commands/help.ts [hunk 1]
```diff
@@ -45,7 +47,7 @@ export default class HelpCommand extends NeedleCommand {
 		const commandsEmbed = await this.getCommandsEmbed(member, channel, showAll);
 
 		await context.interaction.reply({
-			content: "Need more help with Needle? Join us in the [support server](https://discord.gg/8BmnndXHp6)!",
+        		content: `Need more help with Needle? Join us in the [support server](${SUPPORT_SERVER_URL})!`,
```
Comment (MarcusOtter) [inline src/commands/help.ts:50]:
```suggestion
			content: `Need more help with Needle? Join us in the [support server](${SUPPORT_SERVER_URL})!`,
```

**Review by MarcusOtter — COMMENTED** [review]
Thanks 😊 Just need an indentation fix

File: src/commands/help.ts [hunk 2]
```diff
@@ -23,6 +23,8 @@ export default class HelpCommand extends NeedleCommand {
 	public readonly name = "help";
 	public readonly description = "See Needle's commands";
 	public readonly category = CommandCategory.Info;
+	const EMBED_COLOR = "#2f3136";
+	const SUPPORT_SERVER_URL = "https://discord.gg/8BmnndXHp6";
```
[the remark this answers is not in the document]
Comment (EdvinAldi) [reply src/commands/help.ts]: fixed ❤️

**Comment by MarcusOtter** [comment]
Ty ❤️

**Pull request merged**